### DISCUSSION TITLE
Not picking up Barred owl though the log looks like it hears and identifies it

### DISCUSSION THREAD

**Discussion by SharonD214** [mcguirepr89/BirdNET-Pi#967] [Q&A] [answered]
Any ideas why they are not being saved?
I've attached the log...

Thanks
Sharon
![ScreenHunter 70](https://github.com/mcguirepr89/BirdNET-Pi/assets/2545250/ca0f696c-d600-47f9-b6b1-fd7191af57c9)

**Comment by waynecarnold** [comment, marked as answer]
I had a similar issue. Seems the app can pick up birds but will not post them unless part of the list of applicable birds based on the species frequency threshold setting. You can check which birds are applied to your settings in the tools, basic settings page. In order to get the barred owl included in my list i had to make the setting 0.001. Much lower than the default.

**Reply by SharonD214** [reply]
Thanks so much I'll give it a try and hope I hear them again tonight, I could hear them inside with the door closed so was at a loss why they weren't getting picked up on the microphone outside. Moved down to .001 and now it shows on the list.  Barred Owls are pretty common this time of year, surprised I had to set it so low, got the great horned owls on the list now too, I know those are out there.

**Reply by waynecarnold** [reply]
Yes my thoughts as well. Setting it lower brought in many of the common owls in my area. On birdweather.com i am the station called elliot lake

**Reply by SharonD214** [reply]
Great!  I'll check out your birds, I'm Westborough

**Comment by ccle610** [comment]
And so it is also at my side:

I have some owls in the garden that are calling very loudly at the moment.
BirdnetPi shows a blackbird as the last detection around 10 pm, after that nothing more. You can follow the detections in the log:

23:26:41---RECORDING_LENGTH set to 15
23:26:43---BirdNET-Pi/birdnet/bin/python3 BirdNET-Pi/scripts/analyze.py --i BirdSongs/June-2024/22-Saturday/2024-06-22-birdnet-23:26:27.wav --o BirdSongs/June-2024/22-Saturda
y/2024-06-22-birdnet-23:26:27.wav.csv --lat 51.XX --lon 13.XX --week 24 --overlap 0.0 --sensitivity 1.25 --min_conf 0.7 --birdweather_id "IN_USE"
23:26:45---ANALYZING AUDIO... DONE! Time 6.9 SECONDS
23:26:45---WRITING RESULTS TO BirdSongs/June-2024/22-Saturday/2024-06-22-birdnet-23:26:12.wav.csv ... DONE! WROTE 0 RESULTS.
23:26:45---READING AUDIO DATA... DONE! READ 5 CHUNKS.
23:26:45---Date-time: 2024-06-22 23:26:27
23:26:51---0.0;3.0-('Asio otus_Waldohreule', 0.93166083)
23:26:51---3.0;6.0-('Asio otus_Waldohreule', 0.98741525)
23:26:51---6.0;9.0-('Asio otus_Waldohreule', 0.9352277)
23:26:51---9.0;12.0-('Asio otus_Waldohreule', 0.9263579)
23:26:51---12.0;15.0-('Asio otus_Waldohreule', 0.9899182)
23:26:57---Files loaded

I'll try to set frequency threshold setting to 0.005 as mentined above and will be back then . . . hopefuly the owls do appear in the statistics.
Thank you for reading, best regards HaJue

**Reply by ccle610** [reply]
And  . . . hurray !!! They are not longer annonymous . . . it works !!!
Thank you for this hint,  waynecarnold . . . Apropos annonymous ;-)

Birdweather Station # 990 - BirdNET-Pi - Bötzen, Germany - BirdNET-Pi

**Reply by rbrtgg** [reply]
I've had similar issues with owls in particular and am changing the default value to .005 with hopes of detecting them  My main reason for finding this post had to do with bushtits though.  They have been one of the noisiest birds in my back yard for quite a while and then a couple of weeks ago there were no detections on my pi but they were still being detected on the PUC on the other side of the house.  I borrowed a friend's pi with BirdNET on it and recorded for a few hours.  Hers showed the bushtits but not the violet green swallows that mine was detecting.  I changed the default values on both of them and got both birds.
There are a couple of things that I don't understand about all of this.  One is what's the downside of changing the default?  The other is how could a very common and relatively noisy bird like the bushtit in my backyard suddenly no longer be detected?

Bob - BirdNET-pi 284 and PUC 2253

PS - this is my first Github comment so hopefully it is correctly added....
